src/v/security/authorizer.h

```diff
@@ -94,6 +94,9 @@ class authorizer final {
         }
 
         if (acls.empty()) {
+            if (operation == acl_operation::idempotent_write) {
+                return true;
+            }
```

Comment:
looking at the KIP it sounds like we can avoid this specific check that is performed for all operations, and switch checks for `idempotent_write` to be plain `write` and then get rid of `idempotent_write`?

Reply:
I think I'm not familiar with ACLs good enough. It seems in KIP-679 they add new resource for `write`: `ANY` and it isn't supported in Redpanda, is it?

Adding a check for `idempotent_write` was the fastest I came up with to mitigate the problem.

Reply:
ahh ok makes sense thx